pytest_invenio/user.py

```diff
@@ -52,6 +52,7 @@ def create(self, app, db):
                 user.user_profile = self._user_profile
             if self._preferences is not None:
                 user.preferences = self._preferences
+            datastore.commit()
```

Comment:
Since we are commiting the (scoped) session through the datastore (and also creating the user there), is the `begin_nested` in line 39 needed?